# Create awesome lists with Manubot
[![Build Status](https://travis-ci.com/agitter/manubot-awesome-list.svg?branch=master)](https://travis-ci.com/agitter/manubot-awesome-list)

A proof of concept for using [Manubot](http://manubot.org) to automate awesome lists.
An awesome list is a themed list of resources in the `README.md` file in the `master` branch of a GitHub repository.

In this repository, `README.md` is created via continuous integration and should not be edited directly.
Edit `README-BASE.md` to update this text.
Update the reference lists in the `content` directory to add new sections or references.
This is only a proof of concept that is not robust against errors in the scripts or merge conflicts.

The `.travis.yml`, `deploy.sh`, and `environment.yml` files were derived from <https://github.com/manubot/rootstock> (CC0 1.0 license).
## Manubot
1. **Open collaborative writing with Manubot**  
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter  
(2019-07-09) <https://greenelab.github.io/meta-review/>

2. **Opportunities and obstacles for deep learning in biology and medicine**  
Travers Ching, Daniel S. Himmelstein, Brett K. Beaulieu-Jones, Alexandr A. Kalinin, Brian T. Do, Gregory P. Way, Enrico Ferrero, Paul-Michael Agapow, Michael Zietz, Michael M. Hoffman, … Casey S. Greene  
*Journal of The Royal Society Interface* (2018-04-04) <https://doi.org/gddkhn>  
DOI: [10.1098/rsif.2017.0387](https://doi.org/10.1098/rsif.2017.0387) · PMID: [29618526](https://www.ncbi.nlm.nih.gov/pubmed/29618526) · PMCID: [PMC5938574](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5938574)

3. **Python utilities for Manubot: Manuscripts, open and automated: manubot/manubot**  
Manubot  
(2019-07-18) <https://github.com/manubot/manubot>


## Protein-engineering
1. **Machine-learning-guided directed evolution for protein engineering**  
Kevin K. Yang, Zachary Wu, Frances H. Arnold  
*Nature Methods* (2019-07-15) <https://doi.org/gf43h4>  
DOI: [10.1038/s41592-019-0496-6](https://doi.org/10.1038/s41592-019-0496-6) · PMID: [31308553](https://www.ncbi.nlm.nih.gov/pubmed/31308553)

2. **Batched Stochastic Bayesian Optimization via Combinatorial Constraints Design**  
Kevin K. Yang, Yuxin Chen, Alycia Lee, Yisong Yue  
*arXiv* (2019-04-17) <https://arxiv.org/abs/1904.08102v1>

3. **Unified rational protein engineering with sequence-only deep representation learning**  
Ethan C. Alley, Grigory Khimulya, Surojit Biswas, Mohammed AlQuraishi, George M. Church  
*Cold Spring Harbor Laboratory* (2019-03-26) <https://doi.org/gf48g2>  
DOI: [10.1101/589333](https://doi.org/10.1101/589333)

4. **Navigating the protein fitness landscape with Gaussian processes**  
P. A. Romero, A. Krause, F. H. Arnold  
*Proceedings of the National Academy of Sciences* (2012-12-31) <https://doi.org/f4k8bz>  
DOI: [10.1073/pnas.1215251110](https://doi.org/10.1073/pnas.1215251110) · PMID: [23277561](https://www.ncbi.nlm.nih.gov/pubmed/23277561) · PMCID: [PMC3549130](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3549130)


## Single-cell-pseudotime
1. **A comparison of single-cell trajectory inference methods**  
Wouter Saelens, Robrecht Cannoodt, Helena Todorov, Yvan Saeys  
*Nature Biotechnology* (2019-04-01) <https://doi.org/gfxsgd>  
DOI: [10.1038/s41587-019-0071-9](https://doi.org/10.1038/s41587-019-0071-9) · PMID: [30936559](https://www.ncbi.nlm.nih.gov/pubmed/30936559)

2. **An overview of algorithms for estimating pseudotime in single-cell RNA-seq data: agitter/single-cell-pseudotime**  
Anthony Gitter  
(2019-07-19) <https://github.com/agitter/single-cell-pseudotime>

3. **Network Inference with Granger Causality Ensembles on Single-Cell Transcriptomic Data**  
Atul Deshpande, Li-Fang Chu, Ron Stewart, Anthony Gitter  
*Cold Spring Harbor Laboratory* (2019-01-30) <https://doi.org/gft4bb>  
DOI: [10.1101/534834](https://doi.org/10.1101/534834)


